src/panels/DumpPanel.php

```diff
@@ -88,6 +93,22 @@ public function save()
         return $messages;
     }
 
+    public function varDumper($var)
+    {
+        if (is_callable($this->varDumperCallback)) {
+            return call_user_func($this->varDumperCallback, $var, $this);
```

Comment:
That's not valid though; object properties can't be used as variable functions. It would look for a method with the name `varDumperCallback`, and not call the anonymous function of the property.

I could assign it to a variable first, but I think `call_user_func` is more explicit and used by yii2 in many places.